content/en/docs/tasks/traffic-management/egress/egress-gateway/index.md

```diff
@@ -135,80 +135,161 @@ First create a `ServiceEntry` to allow direct traffic to an external service.
 1.  Create an egress `Gateway` for _edition.cnn.com_, port 80, and a destination rule for
     traffic directed to the egress gateway.
 
-    {{< tip >}}
-    To direct multiple hosts through an egress gateway, you can include a list of hosts, or use `*` to match all, in the `Gateway`.
-    The `subset` field in the `DestinationRule` should be reused for the additional hosts.
-    {{< /tip >}}
+{{< tabset category-name="config-api" >}}
 
-    {{< text bash >}}
-    $ kubectl apply -f - <<EOF
-    apiVersion: networking.istio.io/v1alpha3
-    kind: Gateway
-    metadata:
-      name: istio-egressgateway
-    spec:
-      selector:
-        istio: egressgateway
-      servers:
-      - port:
-          number: 80
-          name: http
-          protocol: HTTP
-        hosts:
-        - edition.cnn.com
-    ---
-    apiVersion: networking.istio.io/v1alpha3
-    kind: DestinationRule
-    metadata:
-      name: egressgateway-for-cnn
-    spec:
-      host: istio-egressgateway.istio-system.svc.cluster.local
-      subsets:
-      - name: cnn
-    EOF
-    {{< /text >}}
+{{< tab name="Istio APIs" category-value="istio-apis" >}}
+
+{{< tip >}}
+To direct multiple hosts through an egress gateway, you can include a list of hosts, or use `*` to match all, in the `Gateway`.
+The `subset` field in the `DestinationRule` should be reused for the additional hosts.
+{{< /tip >}}
+
+{{< text bash >}}
+$ kubectl apply -f - <<EOF
+apiVersion: networking.istio.io/v1alpha3
+kind: Gateway
+metadata:
+  name: istio-egressgateway
+spec:
+  selector:
+    istio: egressgateway
+  servers:
+  - port:
+      number: 80
+      name: http
+      protocol: HTTP
+    hosts:
+    - edition.cnn.com
+---
+apiVersion: networking.istio.io/v1alpha3
+kind: DestinationRule
+metadata:
+  name: egressgateway-for-cnn
+spec:
+  host: istio-egressgateway.istio-system.svc.cluster.local
+  subsets:
+  - name: cnn
+EOF
+{{< /text >}}
 
-1.  Define a `VirtualService` to direct traffic from the sidecars to the egress gateway and from the egress gateway
+{{< /tab >}}
+
+{{< tab name="Gateway API" category-value="gateway-api" >}}
+
+{{< text bash >}}
+$ kubectl apply -f - <<EOF
+apiVersion: gateway.networking.k8s.io/v1
+kind: Gateway
+metadata:
+  name: cnn-egress-gateway
+  annotations:
+    networking.istio.io/service-type: ClusterIP
```

Comment:
This seems like the only "special" config needed to create an egress gateway (i.e., no public IP as usual for ingress gateways). Is there anything else that should be configured differently for egress vs. ingress gateways?

cc @howardjohn

Reply:
Should there be a non Istio-specific way to indicate or deduce a Gateway is for egress vs ingress?

Reply:
This looks right to me. Egress and ingress are, IMO, the same thing. You could have 1 gateway doing ingress and egress. Splitting them is mostly an operational convention (and you generally don't want external access to egress)